debvader/debvader.py

```diff
@@ -110,7 +113,7 @@ def deblend_field(net, field_image, galaxy_distances_to_center, cutout_images =
         output_images_mean, output_images_distribution = deblend(net, cutout_images[list_idx], normalised=normalised)  
     if list_idx==[]:
         print('No galaxy deblended. End of the iterative procedure.')
-        return field_image, field_image, np.zeros((field_size,field_size,nb_of_bands)), np.zeros((field_size,field_size,nb_of_bands)), np.zeros((field_size,field_size,nb_of_bands)), cutout_images, output_images_mean, output_images_distribution, np.zeros((len(output_images_mean),2)), list_idx
+        return field_image, field_image, np.zeros((field_size,field_size,nb_of_bands)), np.zeros((field_size,field_size,nb_of_bands)), np.zeros((field_size,field_size,nb_of_bands)), cutout_images, output_images_mean, output_images_distribution, np.zeros((len(output_images_mean),2)), list_idx, 0
```

Comment:
maybe the return value should be simplified.
dict ?!
dataclass ?!

Reply:
Maybe we should also start adding the return variables in the docstring?

Reply:
True. Changed in [7754ad3](https://github.com/BastienArcelin/debvader/commit/7754ad39feb606cfee8e4b0f776da5645eb16258#diff-a5dae2bb532104d1ce10a8b52d5c651b023b7d0d57e87a12c0c6942e0d44cee7) with corresponding changes in the notebook